src/matgl/ext/ase.py

```
@@ -405,8 +405,8 @@ def __init__(
             stress of the atoms
             state_attr (torch.Tensor): State attr.
             stress_weight (float): conversion factor from GPa to eV/A^3
-            ensemble (str): choose from "nve", "nvt", "nvt_langevin", "nvt_andersen", "npt", "npt_berendsen",
-            "npt_nose_hoover"
+            ensemble (str): choose from "nve", "nvt", "nvt_langevin", "nvt_andersen", "nvt_bussi", "npt", 
```

Ruff (0.7.0)

Trailing whitespace

Remove trailing whitespace

(W291)